homeassistant/components/hassio/auth.py

```diff
@@ -82,7 +83,7 @@ async def post(self, request, data):
                 data[ATTR_USERNAME], data[ATTR_PASSWORD]
             )
         except auth_ha.InvalidAuth:
-            raise HTTPUnauthorized() from None
+            raise HTTPUnauthorized(reason=_REASON_INVALID_USER_PASSWORD) from None
```

Comment:
raise HTTPNotFound()

Reply:
Done! Thanks! 👍

Reply:
I had to update a unit test to reflect the new behaviour